### ISSUE TITLE
Error running config for nvim-lspinstall

### ISSUE THREAD

**Issue by Bakermg** [ChristianChiarulli/LunarVim#997]
Updated to the latest version of neovim v0.6.0-dev+27118c6 Build type Release LuaJIT 2.1.0-beta3
install lunarvim with script 

When running nvim  command  getting this error:
![Screen Shot 2021-07-16 at 16 06 06](https://user-images.githubusercontent.com/6689806/126003047-a72132fd-837d-4bbd-a026-cefe28349def.png)

Running PackerCompile does nothing 
Running PackerUpdate closes nvim
Running PackerSync closes nvim

Lunarvim will not load.
Looking for any kind of help or direction
Tried running the overwrite script as well with no luck.

**Comment by rebuilt** [comment]
Does this directory exist on your system?

```
~/.local/share/nvim/site/pack/packer/opt/nvim-lspinstall
```

If not, install it manually

```
cd ~/.local/share/nvim/site/pack/packer/opt/ && git clone https://github.com/kabouzeid/nvim-lspinstall.git
```

Also try removing the packer_compiled file

```
cd ~/.config/nvim && rm -rf plugin
```

**Comment by Bakermg** [comment]
That directory didn't exist so added it manually, and removed the packer_compiles files, still no luck and getting the same error

**Comment by rebuilt** [comment]
Did you make an empty directory or did you clone lspinstall.  The command I provided clones it to that directory

**Comment by Bakermg** [comment]
I execute the command you supplied to clone lspinstall, 
![Screen Shot 2021-07-16 at 16 47 50](https://user-images.githubusercontent.com/6689806/126006970-989b8420-3d37-4b6d-9c9c-c6164c768c03.png)
now from terminal typing nvim getting this error

**Comment by rebuilt** [comment]
Can you try the manual install?  That might give us a better idea of what's failing.  

https://github.com/ChristianChiarulli/LunarVim/wiki/Installation


First follow the instructions to uninstall.

**Comment by Bakermg** [comment]
yeah so I went through all the steps in manual and at this command ```nvim -u $HOME/.config/nvim/init.lua +PackerInstall```
it brought me back to the original error.

**Comment by rebuilt** [comment]
What's the output of 

```
:checkhealth
```

**Comment by Bakermg** [comment]
Everything checks good except for Perl verson being too old
![Screen Shot 2021-07-16 at 17 59 22](https://user-images.githubusercontent.com/6689806/126012513-fba046c7-4298-4833-b01d-c7cce26db7d9.png)
, weird thing is that when running checkhealth it seems to be using your colorscheme.

**Comment by rebuilt** [comment]
The colorscheme doesn't require a packer installed plugin.


Can you try the rolling branch please?

```
LVBRANCH=rolling bash <(curl -s https://raw.githubusercontent.com/ChristianChiarulli/lunarvim/rolling/utils/installer/install.sh)


```

**Comment by Bakermg** [comment]
No luck with that, one thing that I notice after going through the install process the nvim-lspinstall folder was gone and I had added that from earlier comment you had made. I tried to add it again but still with error

**Comment by rebuilt** [comment]
If packer failed to run you're probably missing all your plugin folders.  You'd have to manually install all the plugins.

The only other thing I can think to try is switching to a different version of neovim

**Comment by Bakermg** [comment]
ok will give that a shot. Thanks for all your help, super appreciate it and all your work, I have been using your nvim configs for a long time now, love them.

**Comment by abzcoding** [comment]
@Bakermg can you test this **WARNING: this will remove your `~/.config/nvim` **
```bash
cd /tmp
sudo apt-get install ninja-build gettext libtool libtool-bin autoconf automake cmake g++ pkg-config unzip curl
bash < (curl -s https://raw.githubusercontent.com/ChristianChiarulli/LunarVim/rolling/utils/bin/install-latest-neovim)
curl -s https://raw.githubusercontent.com/ChristianChiarulli/lunarvim/rolling/utils/installer/install.sh | LVBRANCH=rolling bash -s -- --overwrite
```

**Comment by Bakermg** [comment]
Wanted to comment here that I was able to get everything to work by installing neovim with Homebrew and link it. Previously I had used the script that is here to update neovim.

**Comment by rebuilt** [comment]
When you say you previously used the script, do you mean the neovim compilation instructions no longer work for you?
https://github.com/ChristianChiarulli/LunarVim/wiki/Installation

Or do you mean that they used to be in the README and you could no longer find them?

**Comment by Bakermg** [comment]
So what I am saying is I ran into this issue after using this to update neovim
```cd $(mktemp -d)
git clone https://github.com/neovim/neovim --depth 1
cd neovim
sudo make CMAKE_BUILD_TYPE=Release install
cd ..
sudo rm -r neovim```

but instead by using ```brew install --HEAD neovim ```
and link it, everything worked as expected.

**Comment by rebuilt** [comment]
I can confirm.  I tried updating neovim on a laptop that had a beta 0.5 build.  I couldn't upgrade past the old 0.5 build.  I removed the --depth 1 and was able to compile the newest version.  I've updated the instructions on the wiki.  

I don't understand why the '--depth 1' would break anything.  Could it be related to the switch to 0.6 development?  If someone can explain, I'd appreciate it.  

@Bakermg  Thanks for letting us know it no longer worked.
